docs/articles/2023/20231012-normalized_layer_precalc_stats.md

```diff
@@ -0,0 +1,225 @@
+# Introducing a normalized layer and pre-calculated cell and gene statistics in Census
+
+*Published: October 12, 2023*
+
+*By: [Maximilian Lombardo](mailto:mlombardo@chanzuckerberg.com) and [Pablo Garcia-Nieto](pgarcia-nieto@chanzuckerberg.com)*
+
+The Census team is thrilled to announce the introduction of a library-size normalized expression layer and pre-calculated cell and gene statistics. This work is reflected in the Census schema V1.1.0 tailored to empower your single-cell research.
+
+With these additions users can easily:
+
+- Expand their Census query filters to select genes or cells based on the new metadata included, for example selecting only cells with N number of genes expressed.
+- Export the expanded cell and gene metadata for downstream analysis.
+- Export a normalized expression data matrix for downstream analysis.
+
+These features are currently exclusive to the "latest" versions of the Census data release. We invite your feedback as you explore these novel functionalities.
+
+Keep on reading to find out more about these features!
+
+## Description of new data added to Census
+
+All of the following changes were introduced in the Census schema V1.1.0
+
+### Added a new library-size normalized layer
+
+We have introduced a library-size normalized X layer for the RNA measurements of both the human and mouse experiments under `X["normalized"]`.  The normalized layer is built by dividing each value in the raw count matrix by its corresponding row sum (i.e. size normalization).
+
+It is important to note that in order to preserve high precision for the result of a division we would need to encode data with many decimal point values. This can lead to a representation of data that is very large on disk and on memory. As means to compress this matrix we introduced changes that lead to reduced numerical precision and slight deviations from the expected row sum of 1.0.
+
+In addition, to avoid that non-zero values become zero values during the normalization due to the reduced numerical precision, a small sigma value is added during the calculation of the normalized layer.
+
+### Enhanced gene metadata
+
+The `ms["RNA"].var` data frame for both the human and mouse experiments has been enriched with two new metadata fields:
+
+- `nnz` the number of non-zero (nnz) values, effectively the number of cells expressing this gene.
+- `n_measured_obs` the "measured" cells for this gene, effectively the number of cells for which this gene was measured in their respective dataset.
+
+### Enhanced cell metadata
+
+The `obs` data frame for both the human and mouse experiments is now augmented with the following new metadata, allowing users to forego common calculations used in early data pre-processing. For each cell:
+
+- `raw_sum` the count sum derived from `X["raw"]`.
+- `nnz`: the number of non-zero (nnz) values, effectively the number of genes expressed on this cell.
+- `raw_mean` the average counts.
```

Comment:
these names appear to be wrong - they should be raw_mean_nnz, etc.

Reply:
And given the subtlety of the denominator (ie.. mean of explicitly stored values), it may be worth calling that out, or suggesting "for more details, see the schema"

Reply:
vestigial, this article started back in August. Will correct